scout/load/case.py

```diff
@@ -28,7 +28,7 @@ def load_case(adapter, case_obj, update=False):
 
     if existing_case:
         if update:
-            adapter.update_case(case_obj)
+            adapter.update_case(case_obj, keep_date=False)
```

Comment:
And here

Reply:
If we agree that's all, I can now safely remove them.

Reply:
If you want, otherwise it's fine.